### PR TITLE
[dynamo 3.11] fix eval_frame.c debug prints for 3.11

### PR DESCRIPTION
**Pull request by williamwen42**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #96511
* #96509
* #96508
* #96506
* #96505
* #96504
* #96503
* #96501
* __->__ #96500
* #96499

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames @soumith @anijain2305 @Xia-Weiwen @desertfire

### CODE REVIEW

**Review by jansel — APPROVED** [review]

File: torch/csrc/dynamo/eval_frame.c [hunk 1]
```diff
@@ -652,14 +655,22 @@ static PyObject* _custom_eval_frame(
...
+      _PyInterpreterFrame_LASTI(frame));
+  #else
+  DEBUG_TRACE(
+      "begin %s %s %i %i %i",
       name(frame),
       PyUnicode_AsUTF8(frame->f_code->co_filename),
       frame->f_lineno,
       frame->f_lasti,
-      frame->f_iblock,
-      frame->f_executing);
```
Comment (albanD) [inline torch/csrc/dynamo/eval_frame.c:662]: Why remove this?

**Review by albanD — APPROVED** [review]
SGTM

File: torch/csrc/dynamo/eval_frame.c (hunk 1 above)
Comment (williamwen42) [reply torch/csrc/dynamo/eval_frame.c:662]: `f_executing` is not present in 3.10.

**Pull request closed without merging**